wasmsdk/jsbridge/input.go

```diff
@@ -158,3 +162,20 @@ func jsValueToFloat64(jv js.Value) reflect.Value {
 
 	return reflect.ValueOf(i)
 }
+
+func jsValueToStringSlice(jv js.Value) reflect.Value {
+	var list []string
+
+	if jv.Truthy() {
```

Comment:
`Truthy` didn't cover if js.Value is type of Array specifically. Hence, we need to call `isArray` function to be precised.